package-overrides/github/cloudinary/cloudinary_js@1.0.25.json

```diff
@@ -0,0 +1,16 @@
+{
+  "main": "js/jquery.cloudinary",
+  "map": {
+    "jquery.ui.widget": "./js/jquery.ui.widget"
+  },
```

Comment:
I don't think this map is necessary? Or is there an AMD file in this repo that can be loaded?